uber/config.py

```diff
@@ -112,12 +112,16 @@ def get_attendee_price(self, dt=None):
                 badges_sold = 0
             else:
                 dt = sa.localized_now()
-                # this is a database query and very expensive
-                badges_sold = self.BADGES_SOLD
 
             for day, bumped_price in sorted(self.PRICE_BUMPS.items()):
                 if (dt or datetime.now(UTC)) >= day:
                     price = bumped_price
+                    # If we set a price during the event, it should be used regardless of badge sales
+                    if c.EPOCH >= day >= c.ESCHATON:
+                        return price
+
+            # this is a database query and very expensive
+            badges_sold = self.BADGES_SOLD
```

Comment:
this breaks c.HARDCORE_OPTIMIZATIONS_ENABLED which wants to force/fake badges_sold to zero to avoid the DB query.

though, that optimization is kind of a hack anyway, so either we should fix it here or just kill that entire concept from this section.  either way is OK with me.